pkg/printer/table.go

```diff
@@ -21,6 +21,18 @@ func NewTableSection(out io.Writer) *tablewriter.Table {
 	return table
 }
 
+func PrintTableParameterSet(out io.Writer, params [][]string, headers ...string) error {
```

Comment:
Reusing the same method that was used to print before would require a lot of refactoring that would make the function unreadable. Also being that the other method uses it's slice value for the length of the table values (rows) it would always be 1 and append all values on the same row which didn't prove to be what we wanted.  
_(1 ParameterSet can have multiple values, this would use the 1 parameter set as the length and changing that would mess up other methods that call this.)_

Also, using this method doesn't break existing functionality with other parts of the code that uses it.